app/src/main/java/com/example/drawingactivity/CustomColorViewModel.kt

```diff
@@ -29,10 +31,20 @@ class CustomColorViewModel(
         updateViews()
     }
 
+    private var viewModelDrawnBitmap: Bitmap? = null
+
+    fun saveUserBitmap(b: Bitmap?) {
+        viewModelDrawnBitmap = b
+    }
+
+    fun fetchUpdate() {
+        updateViews()
```

Comment:
Why not just put the contents of `updateViews` here?

Reply:
Removed since I just can call the updateViews function from the lambda this was called from and there is functionally no difference.